src/views/connected/coordinateur/index.js

```diff
@@ -18,18 +18,46 @@ export default function Coordinateur() {
         <div className="fr-grid-row fr-grid-row--center fr-my-15w">
           <div className="fr-col--offset-1 fr-col-10">
             <h3>Bienvenue sur votre tableau de pilotage,</h3>
-            <p>Cet espace vous permet de visualiser l&rsquo;ensemble des informations des CnFS que vous coordonnez ainsi que celles de leur structure,
-              de t&eacute;l&eacute;charger les donn&eacute;es statistiques des CnFS ou encore consulter toute la documentation li&eacute;e au dispositif.</p>
+            <p>Nous sommes ravis de vous accueillir sur votre nouvel espace de pilotage.</p>
+            <p>
+              Nous vous proposons une nouvelle interface de suivi et pilotage des Conseillers num&eacute;rique Coordinateur que vous coordonnez.
+              Ce nouvel espace permet de&nbsp;:
+            </p>
+            <div>
+              <ul>
+                <li>Visualiser le liste des conseillers coordinateurs  que vous coordonnez</li>
+                <li>Consulter les informations des conseillers coordinateurs (activité, statistiques…)</li>
+                <li>Consulter les informations des conseillers coordinateurs (activit&eacute;, statistiques…)</li>
+                <li>Suivre les statistiques territoriales et nationales du dispositif</li>
+              </ul>
+                Et bien plus encore...
+            </div>
+            <br />
+            <p><b>Vous avez des questions&nbsp;?</b></p>
+            <div>
+                L&rsquo;&eacute;quipe d&rsquo;animation se tient &agrave; votre disposition&nbsp;:
+              <br />
+              <ul>
+                <li>Par t&eacute;l&eacute;phone au num&eacute;ro suivant&nbsp;:
+                  <b> 01 58 50 89 42 </b>du <b>lundi</b> au <b>jeudi</b> de <b>09h30 &agrave; 12h00</b> et le <b>mercredi de 14h00 &agrave; 16h00.</b></li>
```

Comment:
Plus d'actu `Par téléphone au numéro suivant : 01 58 50 89 42 le lundi toute la journée et le mercredi et jeudi matin de 9h30 à 12h00`

Reply:
Hello, c'est acté ou pas?parce que j'ai pris le wording noté dans le ticket